Service/NotificationManager.php

```diff
@@ -0,0 +1,170 @@
+<?php
+  /**
```

Comment:
it's a place for license if any. Better place to declare your authority is above the class declaration. There is a special [`@author` annotation for that](http://docs.phpdoc.org/references/phpdoc/tags/author.html).